SPECS/json-c/json-c.spec

```diff
@@ -59,6 +58,9 @@ make %{?_smp_mflags} test
 %{_libdir}/pkgconfig/%{name}.pc
 
 %changelog
+* Fri Feb 9 2024 <elainezhao@microsoft.com> - 0.17-1
```

Comment:
minor issue, but can you change the date from `9` to `09`. I believe the tooling will generate a warning otherwise

Reply:
Also, can you add your name before your email address

Reply:
sure